src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -735,16 +735,22 @@ public class Views : RevitDropDownBase
         protected override SelectionState PopulateItemsCore(string currentSelection)
         {
             Items.Clear();
-            //find all views in the project
-            var fec = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
-            var views = fec.OfClass(typeof(View)).ToElements();
+            // find all views in the project
+            // exclude <RevisionSchedule>
+            var views = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(View))
+                .Where(x => !x.Name.Contains('<'))
+                .ToList();
             
-            //there must always be at least 1 view in a Revit document, so we can exclude the empty list check
+            // there must always be at least 1 view in a Revit document, so we can exclude the empty list check
             foreach (var v in views)
             {
                 Items.Add(new DynamoDropDownItem(v.Name, v));
             }
-            return SelectionState.Restore;
+            Items = Items.OrderBy(x => x.Name).ToObservableCollection();
+            SelectedIndex = 0;
```

Comment:
yes, when you drop a node on canvas it will have automatically selected the first item like so: 

![image](https://cloud.githubusercontent.com/assets/529781/18812643/f1f467f8-82a8-11e6-9d1c-c3a8f9193c0e.png)

but if selection state is set to restore it will look like this: 

![image](https://cloud.githubusercontent.com/assets/529781/18812654/3e5a1980-82a9-11e6-90ef-2769905ec5a1.png)